### PR TITLE
add sync condition for privacy tests

### LINKED ISSUE #6402: privacy ATs failing because tx pool not enabled during sync
think we need a similar fix as was done here for permissioning ATs https://github.com/hyperledger/besu/pull/6384

Note privacy ATs currently running on nightly CI only (not on PRs)

example failure here https://app.circleci.com/pipelines/github/hyperledger/besu/26334/workflows/9ca2d46b-92df-4646-9e7d-fe9153826766/jobs/173226 with error "Transaction pool not enabled"

### PR DESCRIPTION
**Pull request by macfarla**
addresses flakiness of privacy ATs since #6302 

fixes #6402 

cherry-picked from #5968 - kudos to @gtebrean

### CODE REVIEW

**Comment by macfarla** [comment]
privacy ATs run with this code https://app.circleci.com/jobs/github/hyperledger/besu/173800 - you can see the output is improved but that PR has other flakiness hence isolating it into this separate PR

**Comment by macfarla** [comment]
this will fix `Message: Transaction pool not enabled` errors [example](https://app.circleci.com/pipelines/github/hyperledger/besu/26731/workflows/afc986d1-c4ce-4458-8350-d139756d848b/jobs/176610)

File: acceptance-tests/tests/src/test/resources/ibft/ibft.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "chainId": 4,
     "berlinBlock": 0,
     "ibft2": {
-      "blockperiodseconds": 1,
+      "blockperiodseconds": 3,
```
[the remark this answers is not in the document]
Comment (gtebrean) [reply acceptance-tests/tests/src/test/resources/ibft/ibft.json]: no, those were part of testing the ci for for the other PR, I reverted them

**Review by pinges — APPROVED** [review]
LGTM

**Pull request merged**